components/isceobj/Sensor/ALOS.py

```diff
@@ -25,7 +25,12 @@
 # Author: Walter Szeliga
 #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-
+#********************************************************************************
+#This program has been upgraded to handle the ALOS-1 PRF change issue.
+#
+#Cunren Liang, 12-December-2019
+#California Institute of Technology, Pasadena, CA
```

Comment:
Similar to authorship info, change notes should be in the git log if significant. Adding these remarks to each file can quickly get out of hand.

Reply:
Yes, same as first comment. These codes for reading ALOS-1 data are originally from UCSD GMTSAR package and were added to ISCE by early ISCE developers. I updated these codes to handle ALOS-1 PRF change issue. Since these codes are already in ISCE, I had to do my job based on these codes. However, Mark was concerned with copyright issues, so I added these comments to explicitly state that we only modified the codes, but original authors are not us.

Reply:
I don't mean to remove the existing authorship info (which would otherwise be lost, since it appears to predate our source control) but to not add further author info for modifications, since git already retains attribution for those modifications.

Similarly, I think the comment
>This program has been upgraded to handle the ALOS-1 PRF change issue

is better suited as a commit message, since future devs will have to look at the git history anyway to determine what specific changes you made.

Reply:
OK, all removed.